scripts/localstack/setup-s3.sh

```diff
@@ -1,2 +1,4 @@
 #!/usr/bin/env bash
-awslocal s3 mb s3://bucket
+awslocal s3 mb s3://admin
+awslocal s3 mb s3://alert-api-1
+awslocal s3 mb s3://alert-api-2
```

Comment:
That could work, until now what I've done:
- let's say the bare docker compose is to run everything locally (assuming you only have your single laptop, or server)
- the dev one is the most temporary, I'll rename it to "test" because it's actually used for tests so we don't preserve any data, everything's wiped clean

So we can keep this for the test, I'll handle that later when I clean compose files